boards/arm/ks_zion/doc/index.rst

```diff
@@ -0,0 +1,127 @@
+.. _ks_zion_board:
+
+
+Kernelspace Zion
+################
+
+.. contents::
+
+
+Overview
+========
+
+The Kernelspace Zion is a small generic development board, featuring an
+ARM Cortex-M4 STM32F303K8 MCU with the minal required hardware layout
+for beginners to get started quickly. It's a simple open-hardware board
+that has been thought to be assembled easily at home.
+
+Specifications, schematic and pcb layout are available at the related
+`Zion page <https://kernel-space.org/hardware/zion.html>`_
+
+In brief:
+
+* STM32 microcontroller in LQFP32 package
+* GPIO pads all around (JXX) for a generic/quick usage
+* I2C display
+* Board power supply: through USB connector, 5 V supply voltage
+* Reset push button
+* One generic LEDs
+* JTAG header
+* Optional external crystal
+
+.. image:: img/zion.webp
+     :align: center
+     :alt: Zion
+
+
+Hardware
+========
+
+- STM32F303K8 in LQFP32 package
+- ARM |reg| 32-bit Cortex |reg| -M4 CPU with FPU
+- 72 MHz max CPU frequency
+- VDD 5.0V
+- 64 KB Flash
+- 12 KB SRAM
+- GPIO with external interrupt capability
+- 2x12-bit ADC
+- 3x12-bit D/A converters
+- 7 channel DMA controller
+- 22 PWM channels
+- SPI (1)
+- I2C (1)
+- USART (3)
+- CAN (1)
+- RTC
+- General Purpose Timers (5)
+- Infrared transmitter
+- I2C display
+
+
```

Comment:
you're missing a list of "Supported Features", see other boards for how it may look like